### ISSUE TITLE
Setting to only show test results and omit request output 

### ISSUE THREAD

**Issue by lcptrs** [alisa101rs/dothttp#8]
**Is your feature request related to a problem? Please describe.**
While it can be useful to see the request output directly in the terminal, it is just populating the terminal which leads to some problems:
- making logs a lot larger and harder to navigate. Especially anoying in a pipeline
- if a test fails, you have to search the log or the .http file in order to find the failing request

**Describe the solution you'd like**
Having a mode / option flag, that enables a summary of the test result. For example something like, ijhttp:
![grafik](https://github.com/alisa101rs/dothttp/assets/11163056/611c11c6-6595-4873-a373-27bd054d0b9e)

**Addition context**
The script I've used for testing
```
GET https://pokeapi.co/api/v2/berry

> {%
  client.test("mock error", () => {
    client.assert(response.status === 201, "Not a real error :)")
  })
%}


###

GET https://pokeapi.co/api/v2/pokemon?offset=20&limit=20
```

**Comment by alisa101rs** [comment]
Could it be solved with --request and --response format? 
Currently existing feature:
```
GET https://httpbin.org/get

> {%
    client.test("mock error", () => {
        client.assert(response.status === 201, "Not a real error :)")
    })
%}

###

GET https://httpbin.org/get

> {%
    client.test("mock error", () => {
        client.assert(response.status === 203, "Not a real error :)")
    })
    client.test("another error", () => {
        client.assert(response.status === 204, "Maybe a real error?")
    })
%}
```

Running with `--request-format="%R\n"` and `--response-format=%T`
```shell
❯ dothttp --request-format="%R\n" --response-format=%T request.http
GET https://httpbin.org/get
Test `mock error`: FAILED with "Assertion failed: Not a real error :)"
GET https://httpbin.org/get
Test `another error`: FAILED with "Assertion failed: Maybe a real error?"
Test `mock error`: FAILED with "Assertion failed: Not a real error :)"
Error: failed tests mock error, another error, mock error

```

**Comment by lcptrs** [comment]
Sorry for the late response:
Nice to see, that something similar can already be achieved. I'd still like the idea of having a `--ci` or `--aggregated` (or whatever name fits best) argument, since I don't think the format parameters are really obvious for a common case. 

So how about implementing it as an alias for `--request-format="%R\n" --response-format=%T`. We could always revisit this argument and change the implementation to make it more useful, e.g. a result line, like
`2 requests completed, 1 failed test`

**Comment by alisa101rs** [comment]
Sorry for long delay! Added support for that case via #17, published in v0.9.0.
Hope it is what you were looking for!

Example:
```
❯ dothttp --format ci requests.http
┌───────────────┬────────────────────┬───────────────┬────────┐
│ File          │ Request            │ Test          │ Result │
├───────────────┼────────────────────┼───────────────┼────────┤
│ requests.http │ Request to get     │ mock error    │ FAILED │
│ requests.http │ Successful request │ mock error    │ PASSED │
│ requests.http │ #3                 │ another error │ FAILED │
│ requests.http │ #3                 │ mock error    │ FAILED │
└───────────────┴────────────────────┴───────────────┴────────┘
3 requests completed, 2 have failed tests

Process finished with exit code 1
```
